cmd/nerdctl/image_convert.go

```diff
@@ -67,6 +68,14 @@ func newImageConvertCommand() *cobra.Command {
 	imageConvertCommand.Flags().Bool("zstdchunked", false, "Use zstd compression instead of gzip (a.k.a zstd:chunked). Should be used in conjunction with '--oci'")
 	// #endregion
 
+	// #region nydus flags
+	imageConvertCommand.Flags().Bool("nydus", false, "Convert an OCI image to Nydus image. Should be used in conjunction with '--oci'")
+	imageConvertCommand.Flags().String("nydus-builder-path", "nydus-image", "The nydus-image binary path, if unset, search in PATH environment")
+	imageConvertCommand.Flags().String("nydus-work-dir", "/var/cache", "Work directory path for image conversion")
```

Comment:
What happens if non-root user without write access to `/var` runs this command?

Reply:
It will fail and the error output will tell the user that he needs write access to the working dir.

Reply:
This command is supported by non-root users, but the non-root users should explictly specify another working dir for this command.

Reply:
This should be supported for non-root users even without extra flags. Maybe nerdctl data root dir can be used.

Reply:
OK. I'll fix it.